package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "main": "src/index.js",
   "scripts": {
-    "lint": "eslint . && prettier -c .",
+    "lint": "eslint . && prettier -c . && dot-only-hunter test",
```

Comment:
I think we should separate this, it's kinda unrelated. Or just prepend it in the `test` script

Reply:
I think it's part of linting phase, but will move it to "tests", I'll get the same benefits their as well.